CONTRIBUTING.md

```diff
@@ -53,7 +53,10 @@ Icons from different sources (_icon sets_) can be used in the tagging schema. He
 ![Screenshot of a preset in iD with the information details open.](https://github.com/openstreetmap/id-tagging-schema/assets/111561/13549318-cd7c-4dd1-9948-7a2d84662f04)
 
 iD and other tools provide users with a way to learn more about the main tag of a preset. It is important to provide good information in this information panel. Here are a few notes on how to do this:
-- Does your tag have a Wikibase entry? Click the small pencil icon next to the text to open the Wikibase item on the OSM wiki. Improve this wording if needed. If the Wikibase item is missing, [learn more about how to add it in "Current methods for creating new items"](https://wiki.openstreetmap.org/wiki/Data_items#Item_creation_process).
+- Does your tag have a Wikibase entry? Click the small pencil icon next to the text to open the Wikibase item on the OSM wiki. Improve this wording if needed.
+  - If the Wikibase item is missing, [learn more about how to add it in "Current methods for creating new items"](https://wiki.openstreetmap.org/wiki/Data_items#Item_creation_process).
+  - iD queries the `Tag:<key>=<value>` page or `Key:<key>` page and prefers the more specific variant. It then uses "wiki sitelinks" to get the data item from the page. These can be set [here](https://wiki.openstreetmap.org/wiki/Special:SetSiteLink//wiki), see the documentation in link above.
```

Comment:
I think I never did this before. Lets double check first if this is actually needed for the lookup … or only for the Wiki-Relation part.

Reply:
@harahu you added wiki data items for your last two PRs, did you add those SiteLinks described here? Or did it work with just the data item with the right values?